### ISSUE TITLE
Section 508 applicability

### ISSUE THREAD

**Issue by AnnMarieDavis** [Section508Coordinators/DHSA-TT-200-A#2]
Some examples may not clearly be U.S. government websites where Section 508 applies. May need disclaimer at beginning of  course:  Determining Section 508 applicability is not covered in this training. Assume Section 508 applies for all illustrative examples in this training.

**Comment by drewnielson** [comment]
I could go either way on this one. For one thing, more and more companies are getting hit with lawsuits where plaintiffs argue that Section 508 standards or WCAG apply via ADA, and judges are ruling in favor of applying 508 standards, even for private companies. I also think that it probably doesn't matter if 508 technically applies when we are simply using pages for examples.

**Comment by drewnielson** [comment]
@KSmithO , what is your opinion?

**Comment by KSmithO** [comment]
I don't think it really matters here-we are showing them how to test-who cares where the examples come from- not to decide applicability (not that most would catch that anyways). I suspect this won't be noticed but to CYA we can put a disclaimer at the beginning with very little effort. Al also was super big on not showing explicit government agency failures in our failing examples b/c of optics. either way, I don't think this is a show stopper.

**Comment by AnnMarieDavis** [comment]
Agreed - this is not really needed.  Some language was added.
